raiden/raiden_service.py

```diff
@@ -360,8 +360,8 @@ def start(self):
         self.alarm.first_run()
 
         # The transport must not ever be started before the alarm task first
-        # run, because it's this method which synchronizes the node with the
-        # blochain, including the channel's state (if the channel is closed
+        # ran, because it's this method which synchronizes the node with the
```

Comment:
I would either say:
`The transport must not ever be started before the alarm task's first run`
or
`The transport must not ever be started before the alarm task ran first`

Reply:
Second is wrong. Should be "has ran first". My current suggestion is also fine. Just tell me what you want to approve it.